src/tasks/TaskHelper.ts

```diff
@@ -203,6 +203,25 @@ export function getDefaultContainerName(nameHint: string, tag?: 'dev' | 'latest'
     return `${getValidImageName(nameHint)}-${tag}`;
 }
 
+export function getContainerOsString(options?: NetCoreTaskScaffoldingOptions): string {

```

Comment:
It would make sense for this to be a helper that took a string and returned a normalized OS value instead of dealing with specific scaffolding option types. Especially as this could theoretically apply to more Docker tasks than just .NET.

Reply:
yeah that makes sense to me